lib/buffer.js

```diff
@@ -422,6 +426,16 @@ Buffer.prototype.fill = function fill(val, start, end) {
 };
 
 
+Buffer.prototype.address = function address() {
+  return binding.address(this);
```

Comment:
As a note, both this check and the `HasInstance()` checks are necessary. Because the object must be an instance of `Buffer`, and it must contain external array data.